### DISCUSSION TITLE
Wallet Synced-Connections 0

### DISCUSSION THREAD

**Discussion by kambodza2021** [Chia-Network/chia-blockchain#5331] [Support]
Hi,I have maybe some problem with connection in wallet tab.
I Ploting on 4 PC,on all PC key is the same. On 3 PC i only ploting and on 1 remaining I ploting and farming. 
On 3 PC when ploting in wallet tab under conection I have 1,but on the PC when I ploting and Farming under conection I have 0. That Pc is status synced. 
Is that normal that 0 connection or Not. What to do if that is not ok.

Pc 1(plotting) connection1

Pc2 (plotting) connection1

Pc3 (plotting) connection1

Pc4 (plotting and farming) connection 0

Can I get Chia when is connection 0 on Pc where farming

**Comment by kambodza2021** [comment]
Any help

**Comment by BrandtH22** [comment]
@kambodza2021 , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
